cmd/get/clusters/printer_test.go

```diff
@@ -248,33 +248,33 @@ func Test_printOutput(t *testing.T) {
 	}
 }
 
-func newCAPIV1alpha3Cluster(id, created, release, org, description string, conditions []string) *capiv1alpha3.Cluster {
+func newcapiv1beta1Cluster(id, created, release, org, description string, conditions []string) *capiv1beta1.Cluster {
```

Comment:
I was thinking that maybe we could rename the function name and drop the api version from its name. Something like `newCapiCluster`? WDYT?

Reply:
sure!